### ISSUE TITLE
Track developers need an easy way to reset themselves

### ISSUE THREAD

**Issue by macta** [exercism/exercism#4181]
When testing a track, you need an easy way to reset everything and start again so you can see how things unlock and test the flow through the exercises.

Leaving a track doesn't seem to do this (will submit a new issue on this)

**Comment by jackhughesweb** [comment]
Is this a duplicate of #4182 or a request for new track development tools?

**Comment by macta** [comment]
It is kind of - but I think the intent is different - for dev's I want an easy way to reset and start again cleanly.

If leaving and then coming back did do that - then I guess that would work.

Although, having a command line tool to reset - would make automated testing a possiblity?

**Comment by macta** [comment]
Is there a temporary workaround where I can get may status manually reset for my Pharo track? This combined with what looks like a bug in completing and being stuck in completion status has kind of put me in limbo

**Comment by macta** [comment]
For now I've create a bogus second user - but waiting on the bug fix so I don't need to create a 3rd and 4th user

**Comment by rpottsoh** [comment]
Possible dupe of #4079.

**Comment by cmccandless** [comment]
Python track maintainer here. This would be very helpful.

**Comment by NobbZ** [comment]
I'm not sure if I want to reset my own progress on a regular base to test things on my track.

I'd prefer to impersonate a testuser and then test with his dataset.

**Comment by cmccandless** [comment]
@NobbZ why not a combination approach? Implement the reset feature for maintainers/developers, and use a single dummy account for testing, which can be reset any number of times without altering your own personal progress.

**Comment by macta** [comment]
Hadn’t occurred to me that it might be useful to keep my personal track progress intact - so yes I would want to be able reset a second test account at will.

**Comment by ErikSchierboom** [comment]
> why not a combination approach? Implement the reset feature for maintainers/developers, and use a single dummy account for testing, which can be reset any number of times without altering your own personal progress.

Sounds good! I definitely don't want to reset my own progress :)

**Comment by iHiD** [comment]
You can now reset a track and can update test suites so I'm closing this :)
